### PR TITLE
Make tests optional using a global meson option

### PR DESCRIPTION
**Pull request by vvidic**

### CODE REVIEW

File: login/meson.build [hunk 1]
```diff
@@ -72,19 +73,21 @@ if get_option('pam-glome')
...
-            build_by_default : true, output : [ 'pam_service' ],
-            command : [ 'mkdir', '@OUTPUT@' ])
-        test('pam test', pam_test,
-            env: [ 'LD_PRELOAD=libpam_wrapper.so', 'PAM_WRAPPER=1',
-                'PAM_WRAPPER_SERVICE_DIR=' +
-                join_paths(meson.build_root(), 'login', 'pam_service'),
-                'PAM_GLOME=' +
-                join_paths(meson.build_root(), 'login', 'pam_glome.so') ])
+    if get_option('tests')
+        libpamtest = dependency('libpamtest', required : false)
```
Comment (pkern) [inline login/meson.build:77]: Should we require libpamtest when PAM is wanted and tests are requested?

**Review by pkern — APPROVED** [review]
LGTM

File: login/meson.build (hunk 1 above)
Comment (vvidic) [reply login/meson.build]: Sure, let's give it a try :)

Comment (vvidic) [reply login/meson.build:77]: Ah, no good, Alpine does not have this library. But at least Debian now runs this test ;)

**Comment by pkern** [comment]
Thanks!

**Pull request merged**